adminSiteClient/ExplorerCreatePage.tsx

```diff
@@ -20,17 +20,17 @@ import {
     UNSAVED_EXPLORER_DRAFT,
     UNSAVED_EXPLORER_PREVIEW_QUERYPARAMS,
     ExplorerProgram,
-    EXPLORER_FILE_SUFFIX,
     makeFullPath,
     isEmpty,
 } from "@ourworldindata/explorer"
 import { GitCmsClient } from "../gitCms/GitCmsClient.js"
-import { GitCmsFile, GIT_CMS_BASE_ROUTE } from "../gitCms/GitCmsConstants.js"
-import { AdminManager } from "./AdminManager.js"
+import { GIT_CMS_BASE_ROUTE } from "../gitCms/GitCmsConstants.js"
+import { AdminManager } from "../explorerAdminClient/AdminManager.js"
```

Comment:
We can probably move everything else in `explorerAdminClient` too and just get rid of the directory, right?

Reply:
Done! I had it in "future work", but it wasn't as tricky as I expected.